api/managementpb/checks.proto

```diff
@@ -33,10 +33,35 @@ message SecurityCheckResult {
   map<string, string> labels = 4;
 }
 
+// SecurityCheckState contains check name and status.
+message SecurityCheckState {
+  string name = 1;
+  bool disabled = 2;
+}
+
+// SecurityCheckParams specifies check parameters.
+message SecurityCheckParams {
+  string name = 1;
+  bool enable = 2;
+  bool disable = 3;
+}
+
 message GetSecurityCheckResultsResponse {
   repeated SecurityCheckResult results = 1;
 }
 
+message ListSecurityChecksRequest {}
+
+message ListSecurityChecksResponse {
+  repeated SecurityCheckState checks_states = 1;
+}
+
+message UpdateSecurityChecksRequest {
```

Comment:
We use "Change" there: https://github.com/percona/pmm/blob/PMM-2.0/api/inventorypb/agents.proto
So let's stick with it

Reply:
Done